appium/tests/specs/mock/setup/CheckRefreshKeyFromEKMSecond.spec.ts

```diff
@@ -0,0 +1,59 @@
+import { MockApi } from 'api-mocks/mock';
+import {
+  KeysScreen,
+  MenuBarScreen,
+  SettingsScreen,
+  SetupKeyScreen,
+  SplashScreen,
+} from '../../../screenobjects/all-screens';
+import { ekmPrivateKeySamples } from "../../../../api-mocks/apis/ekm/ekm-endpoints";
+import { CommonData } from "../../../data";
+import BaseScreen from "../../../screenobjects/base.screen";
+import AppiumHelper from "../../../helpers/AppiumHelper";
+
+const goToKeysScreen = async () => {
+  await MenuBarScreen.clickMenuIcon();
+  await MenuBarScreen.clickSettingsButton();
+  await SettingsScreen.clickOnSettingItem('Keys');
+}
+
+describe('SETUP: ', () => {
+
+  it('app auto updates keys from EKM during startup without pass phrase prompt', async () => {
```

Comment:
Let's rename the file too - `CheckRefreshKeyFromEkmWithoutPassPhrasePrompt.spec.ts`